lib/src/main/java/edu/utexas/tacc/tapis/files/lib/models/FileInfo.java

```diff
@@ -17,8 +20,40 @@
  */
 public class FileInfo
 {
-    public static final String FILETYPE_FILE = "file";
-    public static final String FILETYPE_DIR = "dir";
+    public  enum FileType {
+        FILE("file"),
```

Comment:
I added some values here to make "file" and "dir" line up with what we used to have.  I'd prefer to make these different - at bare minimum all caps like our other enums.  I don't know how much grief this would cause current users of the interface though.